### ISSUE TITLE
Omnisharp log not helpful when .NET Core SDK is missing

### ISSUE THREAD

**Issue by richlander** [OmniSharp/omnisharp-vscode#3193]
This is the log I see when the .NET Core SDK is missing on Windows.

https://gist.github.com/richlander/30cd2b81fb20a1cfab2ffa1d54f98285

I know it is missing because I also see this dialog in Code. 

![2019-07-26 10_11_16-Window](https://user-images.githubusercontent.com/2608468/61971613-5ce45800-af94-11e9-8d23-25c05815722f.png)

In this scenario, I'd like to not see the [error about missing the Microsoft.Net.SDK](https://gist.github.com/richlander/30cd2b81fb20a1cfab2ffa1d54f98285#file-osharplog-txt-L46-L61) (which isn't directly actionable) and instead see one about not satisfying the .NET Core SDK requirement (hopefully much higher in log).

In short, the message / call to action in the dialog should be represented in the log and it isn't. The log should provide the same message and link to download the .NET Core SDK as what the dialog uses.
